website/content/api-docs/secret/databases/mssql.mdx

```diff
@@ -38,7 +38,7 @@ has a number of parameters to further configure a connection.
   `max_open_connections` it will be reduced to be equal.
 
 - `max_connection_lifetime` `(string: "0s")` - Specifies the maximum amount of
-  time a connection may be reused. If <= 0s connections are reused forever.
+  time a connection may be reused. If &le; `0s` connections are reused forever.
```

Comment:
this doesn't seem to be rendered correctly i think ?
<img width="1011" alt="Screen Shot 2021-12-17 at 12 02 26 PM" src="https://user-images.githubusercontent.com/35388175/146601410-aaf144f9-e61a-4595-b0d2-adb8968c4eff.png">
?

Reply:
You should be able to see the preview from the Vercel link in the github checks list

Reply:
I'll take a look, because I can't build the documentation using `make build` anymore :/ (I get some error with Next.js)

Reply:
![Screenshot from 2021-12-17 21-24-15](https://user-images.githubusercontent.com/65311/146603721-6ff7b6e9-6026-4e4d-bf2a-646ca15f7a46.png)


:thinking: :thinking: :thinking: :thinking:

Reply:
It looks like it's an issue with the "metro-web" font, if I disable it, the symbol switches to the right shape...

Reply:
I'll simply revert the change, if that helps to merge this PR :+1: